### ISSUE TITLE
Accessibility: Search results display empty image tags without alt-text attributes 

### ISSUE THREAD

**Issue by apeters** [archesproject/arches#10342]
All images should have a meaningful alt-text value assigned to them.  If an image tag is empty (no image) the image tag shouldn't be rendered at all.

**Comment by apeters** [comment]
closed by #10344
